### PR TITLE
Added requester pays option to Mutect2 tasks that access bams

### PR DESCRIPTION
**Pull request by ldgauthier**
gnomAD VCFs are not currently requester pays nor are they expected to be

### CODE REVIEW

**Comment by ldgauthier** [comment]
closes #6830

File: scripts/mutect2_wdl/mutect2_pon.wdl [hunk 1]
```diff
@@ -128,6 +131,7 @@ task CreatePanel {
       File gnomad
       File gnomad_idx
       String? create_pon_extra_args
+      String? gcs_project_for_requester_pays
```
Comment (fleharty) [inline scripts/mutect2_wdl/mutect2_pon.wdl:134]: I don't think this is connected to anything.  Should this argument be used on line 152 and 154?

**Review by fleharty — COMMENTED** [review]
Looks good, just one missing piece I think.

**Review by fleharty — CHANGES_REQUESTED** [review]
@ldgauthier back to you.  Just one little change, I think.

File: scripts/mutect2_wdl/mutect2_pon.wdl (hunk 1 above)
Comment (ldgauthier) [reply scripts/mutect2_wdl/mutect2_pon.wdl]: Indeed.  After I asked the gnomAD team and they said it's not, nor is it ever expected to be requester pays, I forgot to take this out.

**Review by droazen — APPROVED** [review]

**Pull request merged**